operations/sftpgo-authentication-service/internal/config/config_test.go

```
@@ -0,0 +1,151 @@
+
+package config
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func TestLoad_Success(t *testing.T) {
+	// Setup environment variables
+	os.Setenv("INTERNAL_CLIENT_ID", "test_client_id")
```

golangci-lint (2.12.2)

[error] Error return value of `os.Setenv` is not checked

(errcheck)